avm/res/container-instance/container-group/tests/e2e/private/main.test.bicep

```diff
@@ -47,7 +47,7 @@ module nestedDependencies 'dependencies.bicep' = {
 
 @batchSize(1)
 module testDeployment '../../../main.bicep' = [
-  for iteration in ['init', 'idem']: {
+  for iteration in ['init']: {
```

Comment:
Not anymore idempotent?

Reply:
🤦Just like last time :) I forgot to add back...

Reply:
Fixed